### PR TITLE
completed code for Lesson02

### PR DESCRIPTION
**Pull request by lmburu-uw**
Please review completed code for Lesson02, exercise 2.2, grid printer. thanks

### CODE REVIEW

File: students/larry_mburu/Lesson02/fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+            print("FizzBuzz") 
+        if num % 3 == 0: 
+            print(num)
+            print("Fizz")
+        if num % 5 == 0: 
+            print(num)
+            print("Buzz")
+
+if __name__ == '__main__': 
+    fizz_buzz()
```
Comment (charlesrobison) [inline students/larry_mburu/Lesson02/fizz_buzz.py]: Close, but not quite.  You're output should be:
1
2
Fizz
4
Buzz
Etc, all the way to 100

File: students/larry_mburu/Lesson02/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def fizz_buzz(): 
+    for num in range(1, 16): 
+        if (num % 3 == 0 ) and (num % 5 == 0):
```
Comment (charlesrobison) [inline students/larry_mburu/Lesson02/fizz_buzz.py]: Extra space between 0 and )

File: students/larry_mburu/Lesson02/grid_printer.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    for i in range(size):
+        print('|', end='')
+        for row in range(size):
+            print(' ' * cell_size, end='')
+            print('|', end='')
+        print()
+
+if __name__ == "__main__":
+    print_grid(5.5, 5)
```
Comment (charlesrobison) [inline students/larry_mburu/Lesson02/grid_printer.py]: Works well!

**Comment by lmburu-uw** [comment]
Charles,

On the FizzBuzz problem, i'm allowed to resubmit it with the right
corrections? Thanks.

**Comment by charlesrobison** [comment]
Hi @lmburu-uw - yes, just submit any corrections in a new PR and I'll update the grade as needed.  :)

**Pull request merged**